extensions/amp-next-page/1.0/test/test-amp-next-page.js

```diff
@@ -495,7 +494,7 @@ describes.realWin(
           .be.ok;
 
         service.pages_[2].visibilityState_ = VisibilityState.VISIBLE;
-        service.scrollDirection_ = Direction.UP;
+        service.visibilityObserver_.scrollDirection_ = ScrollDirection.UP;
```

Comment:
Ah, brings me back to the dock discussion about a `Direction` enum.

Somehow, somewhere, these should be defined the same. Not important but maybe add a `TODO` to consolidate.